.github/workflows/cicd-main.yml

```diff
@@ -5450,6 +5450,21 @@ jobs:
       SCRIPT: |
         bash tests/collections/llm/bitexact/mixtral/run.sh
 
+  L2_NeMo_2_PTQ_Llama2_FP8:
+    needs: [cicd-test-container-setup]
```

Comment:
Just a friendly reminder to make sure it doesn't get lost once this PR is ready: this job name also needs to be added to `CICD_Nemo_Test` like the others

Reply:
Added